README.md

```diff
@@ -1,3 +1,33 @@
-# Template course for PrairieLearn
+## PrairieLearn Course Repository
+
+### Welcome to PrairieLearn! 
+
+The content for your course is stored within this repository.
+You can see and edit a live version at <https://prairielearn.engr.illinois.edu/>.
+
+### Overview
+
+PrairieLearn stores data for a course using folders and JSON files. 
+Each of these files can be edited directly using [PrairieLearn website's "Files" or "Settings" tab](https://prairielearn.engr.illinois.edu/). 
+The roles of each file and directory are:
+
+- **[infoCourse.json](infoCourse.json)**: Store basic information about the course such as its number and name, e.g. PL 101: Using PrairieLearn.
+- **[courseInstances/](courseInstances/)**: Directory containing per-semester offerings of a course, e.g. Fall 2020 and Spring 2021.
+   - **[courseInstances/Fa20/infoCourseInstance.json](courseInstances/Fa20/infoCourseInstance.json)**: The upcoming Fall 2020 semster is found here. Make sure to set when the course should open and list the course's teaching staff.
+   - **[courseInstances/Fa20/assessments](courseInstances/Fa20/assessments)**: Any assessment such as homework or exams for the Fall 2020 semester will be located here. These assessments are specific to the single-time course offering.
+- **[questions/](questions/)**: All questions created for a course are found here. 
+   - **[questions/template](questions/template)**: Inside of the **[questions/](questions/)** directory, templates containing the best design practices for creating assessment items can be found. We recommend using these templates as a base for creating your own questions.
+
+### Reference Guide
+
```

Comment:
Maybe just something along the lines of "Bookmark or save this reference guide as a cheat-sheet to use while developing questions."

Reply:
Switched the reference guide image size to be miniaturized and opted for the pages to be side-to-side.